README.md

```diff
@@ -98,7 +127,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer...
+Configures/ requires body-parser to be used by the app (body parser takes the data in HTML form and makes it JSON so express can use it). line 2 handles JSON post requests, and line 3 handles form submissions
 ```
```

Comment:
Close. This allows our app to (1) take data in through a form and also (2) take data in as JSON. You do not need one in order to implement the other -- they are separate things. Happy to elaborate if you have any questions about this one.